ext/zip/config.m4

```diff
@@ -1,91 +1,47 @@
 dnl config.m4 for extension zip
 
 PHP_ARG_ENABLE([zip],
-  [for zip archive read/writesupport],
+  [for zip archive read/write support],
   [AS_HELP_STRING([--enable-zip],
     [Include Zip read/write support])])
 
-PHP_ARG_WITH([libzip],
-  [libzip],
-  [AS_HELP_STRING([[--with-libzip[=DIR]]],
-    [ZIP: use libzip])],
-  [yes],
-  [no])
-
 if test "$PHP_ZIP" != "no"; then
+  PKG_CHECK_MODULES([LIBZIP], [libzip >= 0.11])
+  LIBZIP_LIBDIR=`$PKG_CONFIG --variable=libdir libzip`
+
+  dnl Could not think of a simple way to check libzip for overwrite support
+  PHP_CHECK_LIBRARY(zip, zip_open,
+  [
+    PHP_ADD_LIBRARY_WITH_PATH(zip, $LIBZIP_LIBDIR, ZIP_SHARED_LIBADD)
+    AC_DEFINE(HAVE_LIBZIP,1,[ ])
```

Comment:
`PHP_ADD_LIBRARY_WITH_PATH` largely seems to replicate `PHP_EVAL_INCLINE` and `PHP_EVAL_LIBLINE`. It may be worthwhile using the latter macros in a later patch.